module/config/argparser_en.py

```diff
@@ -579,6 +579,26 @@ def choice_list(total):
     # os_semi = os_semi_parser.add_argument_group('os_clear_map', 'Only recommended to use in save zones. To use in normal zones, execute air search manually first.\nUsage: Enter map manually and run\nRecommend to re-check map manually after run', gooey_options={'label_color': '#931D03'})
     # os_semi.add_argument('--enable_os_ash_attack', default=default('--enable_os_ash_attack'), choices=['yes', 'no'], help='Attack ash if beacon data is full', gooey_options={'label_color': '#4B5F83'})
 
+    # ==========OS clear world==========
+    os_world_parser = subs.add_parser('os_world_clear')
+    os_world = os_world_parser.add_argument_group('OS world clear',
+                                                  'Explore all unsafe zones between configured range inclusive and turn into safe\n'
+                                                  'Captains should configure approriately based on current adaptibility numbers and needs',
+                                                  gooey_options={'label_color': '#931D03'})
+    os_world.add_argument('--os_world_min_level', default=default('--os_world_min_level'),
+                          help='No less than 1\n'
+                          'Use same number in both fields for exactly 1 range',
+                          gooey_options={'label_color': '#4B5F83'})
+    os_world.add_argument('--os_world_max_level', default=default('--os_world_max_level'),
+                          help='No higher than 7\n'
+                               'Recommend 4 or lower for single fleet clear, 5 and higher '
+                               'can be difficult with low adaptibility',
+                          gooey_options={'label_color': '#4B5F83'})
+    os_world.add_argument('--enable_os_world_repair', default=default('--enable_os_world_repair'), choices=['yes', 'no'],
+                          help='Enable retreat to nearest azur port for repair if any one ship in fleet below configured threshold',
+                          gooey_options={'label_color': '#4B5F83'})
+    os_world.add_argument('--os_world_repair_threshold', default=default('--os_world_repair_threshold'), help='Limit between 0 and 1', gooey_options={'label_color': '#4B5F83'})
+
```

Comment:
Need to emphasize that user need to clear the story mode in operation siren, buy and use 5k oil item.
Set `choices` argument in `os_world_min_level` and `os_world_max_level` to limit user input. I think any fleets over level 100 with Perseus and port repair is able to claer CL6.

Reply:
Sure, I'll add in a requirements line before usage.

`choices` is easy adjustment and makes sense to limit the user.

Perhaps I should add when acquiring the zones of a certain hazard level to force fleet repair since for example if threshold isn't met after clearing an already relatively difficult zone then that would be bad as it be unlikely they could clear it.
```
for zone in zones:
    if not self.globe_goto(zone, stop_if_safe=True):
        continue
    self.run_auto_search()
    # self.need_repair(self.config.ENABLE_OS_WORLD_REPAIR,
    #                           self.config.OS_WORLD_REPAIR_THRESHOLD)
    handle_fleet_repair(force=True if hazard_level >= 5 else False)
```

Reply:
I think it's not neccessary either. If you do want to check HP, check it after every combat, and stop auto search when lower than threshold.

Reply:
I think there may be 1 issue with checking after every battle especially when in auto search mode. Enemy nodes could be right next to each other and start a battle quite soon before ALAS could press the auto search button. 

It may easily get stuck with perhaps having enough time to detect hp but it already starts a new a battle. Making ALAS think, it can go fleet repair but the option to do that has disappeared now.

`hp_get()` screen capture may be taken during a loading screen or in battle thereby getting inaccurate reads as well.

While this would be easily remedied but I think should still be mentioned obscure zones, you wouldn't want to exit mid-run seeing as the zone disappears when you leave.